ansible/roles/telegraf/meta/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright (C) 2021 Dr. Serge Victor <https://dr.sergevictor.eu/>
+# Copyright (C) 2021 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
You should add collection information so that Ansible can find plugins and modules properly, usually it's formatted as:
```yaml
# Ensure that custom Ansible plugins and modules included in the main DebOps
# collection are available to roles in other collections.
collections: [ 'debops.debops' ]
```